rc4me/run.py

```diff
@@ -1,13 +1,30 @@
+"""TODO - Docstring."""
+
+from pathlib import Path
+from typing import Optional
+import logging
+import click
+
+from rc4me.fetch_rcs import fetch_rcs
 from rc4me.util import copyfiles
 
+logging.basicConfig(level=logging.DEBUG)
+logger = logging.getLogger(__name__)
 
-DESTINATION_DIR = "./"
-SOURCE_DIR = "~/rc4me/"
+RC4ME_HOME = Path.home() / ".rc4me"
 
 
-def main():
-    """Run copyfiles"""
-    copyfiles(SOURCE_DIR, DESTINATION_DIR)
+@click.command()
+@click.argument("repo", type=str)
+@click.argument("dest", required=False, type=str)
```

Comment:
I would make `dest` and option that defaults to `Path.home`
```
@click.option("--dest", default=Path.home(), type=click.Path())
```

Reply:
In general, I rarely use arguments because I think options are a lot clearer, but that's just a style thing.